thorlcr/activities/funnel/thfunnelslave.cpp

```diff
@@ -795,20 +794,36 @@ class CNWaySelectActivity : public CSlaveActivity, public CThorSteppable
                 IThorNWayInput *nWayInput = dynamic_cast<IThorNWayInput *>(cur);
                 if (nWayInput)
                 {
+                    cur->start();
```

Comment:
A nway input (CThorNWayInputSlaveActivity), is really just a delayed container for the selectable inputs. It  does not start of it's [potential] inputs.
There's a comment in CThorNWayInputSlaveActivity::start();
```
        // NB: Whatever pulls this IThorNWayInput, starts and stops the selectedInputs
```
and in CThorNWayInputSlaveActivity::stop()
```
// NB: Whatever pulls this IThorNWayInput, starts and stops the selectedInputs
```

I start the NWay input here, because this activity (like all others) doesn't automatically call PARENT::start to start it's inputs.
Because it wants to only start the selectedInput in the case of a NWay.

So need to started NWay activity to prepare the possibilities and then start the selected input, which may or may not be one from a NWayInput activity upstream.